xgo.go

```diff
@@ -31,8 +31,8 @@ func init() {
 		depsCache = filepath.Join(home, ".xgo-cache")
 		return
 	}
-	if user, err := user.Current(); user != nil && err == nil && user.HomeDir != "" {
-		depsCache = filepath.Join(user.HomeDir, ".xgo-cache")
+	if usr, err := user.Current(); usr != nil && err == nil && usr.HomeDir != "" {
```

Comment:
Why did you change this?

Reply:
This is an unrelated change to the PR, but still should be made. 

`user, err := user.Current()` doesn't make sense as `user` would be overridden. changing it to `usr, err := user.Current();` fixes the problem.

Reply:
Ah I see, makes sense!

Reply:
The current variable name `user` shadows the package import `os/user`. It's bad practice though although a small nit.